vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -0,0 +1,248 @@
+#### Create the descriptors
+1. Adapt the descriptors from the primary tenant for the secondary tenant by editing the names with YOUR-TENANT1-CODE where applicable and the ingress with the additional tenant1 subdomain prefix. The descriptors include the persistent volume discriptor, Keycloak 
+2. Apply the descriptors
+3. Create the API requests which should include the following parameters:
+  * "CDS_PATH" : # the base path of the new CDS 
+  * "KC_URI" : # the url of token service of the tenant1 realm 
+  * "KC_CLIENT_ID" : # the clientId of the reserved client used by the enanto-de-app in tenant1 realm
+  **what is the reserved client???**
+  * "KC_CLIENT_SECRET" : # the secret of that client
+
+4. Create the archive and upload it to the CDS service the resources for YOUR-TENANT1-CODE. 
+
+### Databases 
+Create a single schema for your database that maps all the tables for content, templates, users, groups, widgets, etc. 
+* Add Liquibase DB management option for secondary tenants
+
+### Ingress
+Create and apply the ingress descriptor with the following information.
+``` yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    entando.org/quickstart-appbuilder-path: /app-builder/
+    entando.org/quickstart-de-path: /digital-exchange
+    entando.org/quickstart-pn-3c07adf0-fac54a9f-entando3323-pn-3c07adf0-fa-path: /entando-epc-bootstrap-3c07adf0/app-builder-menu-bff
+    entando.org/quickstart-server-path: /entando-de-app
+  generation: 4
+  labels:
+    EntandoApp: quickstart
+  name: YOUR-TENANT1-CODE-quickstart-ingress
+  namespace: entando
+spec:
+  rules:
+  - host: YOUR-TENANT1-CODE.YOUR-HOST-URL
+    http:
+      paths:
+      - backend:
+          service:
+            name: quickstart-ab-service
+            port:
+              number: 8081
+        path: /app-builder/
+        pathType: Prefix
+      - backend:
+          service:
+            name: quickstart-service
+            port:
+              number: 8080
+        path: /entando-de-app
+        pathType: Prefix
+      - backend:
+          service:
+            name: quickstart-cm-service
+            port:
+              number: 8083
+        path: /digital-exchange
+        pathType: Prefix
+      - backend:
+          service:
+            name: pn-3c07adf0-fac54a9f-entando-app-builder-menu-bff-service
+            port:
+              number: 8081
+        path: /entando-epc-bootstrap-3c07adf0/app-builder-menu-bff
+        pathType: Prefix
+```
+
+### ConfigMap 
+Create and apply a new ConfigMap for the secondary tenant. Then edit the App Engine `entando-de-app` deployment to point to this ConfigMap.
+
+1. Create a new ConfigMap containing the tenant codes, Keycloak, database, CDS and Solr services and ingresses. 
+
+Here is an example: **isn't part in yaml format and part in json format???**
```

Comment:
@pietrangelo, this is the sample file I was told to use here tenants_cm.yaml: https://drive.google.com/drive/u/0/folders/119Y6dyZV35Y6u-Alz2KDKcjupUflivI5

But isn't the format strange?

Reply:
@jyunmitch No, it isn't. This is a configMap which contains a json object definition as its data, so it's correct.